swarm/tools/flow_studio_ui/fragments/10-header.html

```
@@ -41,7 +41,7 @@ <h1>Flow Studio</h1>
       <div class="muted author-only" style="display: flex; align-items: center; gap: 8px;" data-uiid="flow_studio.header.reload">
         <span class="mono">make dev-check</span>
         <button id="copy-dev-check-btn" class="copy-btn" title="Copy command to clipboard" data-uiid="flow_studio.header.reload.copy_btn">Copy</button>
-        <button id="reload-btn" class="fs-button-small" data-uiid="flow_studio.header.reload.btn">Reload</button>
+        <button id="reload-btn" class="fs-button-small" aria-label="Reload Dev Check" data-uiid="flow_studio.header.reload.btn">Reload</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)